src/classes/OPP_OpportunityNaming.cls

```diff
@@ -188,22 +188,24 @@ public with sharing class OPP_OpportunityNaming implements OPP_INaming {
                 }
             }
 
+            String oppAttributionValue = STG_PanelOppNaming_CTRL.convertToAttributionNamingLabel(ons.Attribution__c);
```

Comment:
Here we are introducing an `enum` value to be stored in the Opportunity Naming Setting, ie `Both`, `Contact_Donations` and `Organization_Donations`. Current orgs for English language can have `Both`, `Contact Donations` or `Organization Donations` value while orgs for a language other than English will have their translated value stored in the settings. I have not tested it, but it seems the change will not work for existing orgs. Can you please confirm? Does the NPSP Settings page view and edit work when settings value is 'Contact Donations' for example?

Reply:
That is right, @RadSFDeveloper. However, it converts to label values(from custom settings values) for the logic, so it is consistent for English and non English users. I checked the changes for Contact Donations type for both English and Spanish users, it works fine.